doc/under_sampling.rst

```diff
@@ -347,10 +347,30 @@ place. The class can be used as::
 Our implementation offer to set the number of seeds to put in the set :math:`C`
 originally by setting the parameter ``n_seeds_S``.
 
-:class:`NeighbourhoodCleaningRule` will focus on cleaning the data than
-condensing them :cite:`laurikkala2001improving`. Therefore, it will used the
-union of samples to be rejected between the :class:`EditedNearestNeighbours`
-and the output a 3 nearest neighbors classifier. The class can be used as::
+Neighbourhood Cleaning Rule
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The :class:`NeighbourhoodCleaningRule` is another "cleaning" algorithm. It removes
+samples from the majority class that are closest to the boundary with the minority
```

Comment:
```suggestion
samples from the majority class that are the closest to the boundary formed by the samples of the minority class
```

Reply:
I don't totally understand this sentence. Let me try a modification in a new commit.